spicepod.yaml

```diff
@@ -0,0 +1,104 @@
+version: v1beta1
+kind: Spicepod
+name: spice-oss-docs
+
+datasets:
+  - from: github:github.com/spiceai/docs/files/trunk
+    name: spiceai.docs
+    description: Spice.ai OSS documentation and reference, from https://docs.spiceai.org
+    metadata:
+      reference_base_url: https://docs.spiceai.org/<docs_path>
+    params:
+      github_token: ${secrets:GITHUB_TOKEN}
+      include: "spiceaidocs/docs/**/*.md"
+    acceleration:
+      enabled: true
+      refresh_check_interval: 12h
+      refresh_jitter_enabled: true
+      refresh_jitter_max: 1m
+    embeddings:
+      - column: content
+        use: openai_embeddings
+        column_pk:
+          - path
+  
+  - from: github:github.com/spiceai/samples/files/trunk
+    name: spiceai.samples
+    description: Spice.ai OSS samples
+    metadata:
+      reference_base_url: https://github.com/spiceai/samples/tree/trunk/<sample_path>
+    params:
+      github_token: ${secrets:GITHUB_TOKEN}
+      include: "**/*.md"
+    acceleration:
+      enabled: true
+      refresh_check_interval: 12h
+      refresh_jitter_enabled: true
+      refresh_jitter_max: 1m
+    embeddings:
+      - column: content
+        use: openai_embeddings
+        column_pk:
+          - path
+
+  - from: github:github.com/spiceai/quickstarts/files/trunk
+    name: spiceai.quickstarts
+    description: Spice.ai OSS quickstarts
+    metadata:
+      reference_base_url: https://github.com/spiceai/quickstarts/tree/trunk/<quickstart_path>
+    params:
+      github_token: ${secrets:GITHUB_TOKEN}
+      include: "**/*.md"
+    acceleration:
+      enabled: true
+      refresh_check_interval: 12h
+      refresh_jitter_enabled: true
+      refresh_jitter_max: 1m
+    embeddings:
+      - column: content
+        use: openai_embeddings
+        column_pk:
+          - path
+
+embeddings:
+  - from: openai
+    name: openai_embeddings
+    params:
+      openai_api_key: ${secrets:OPENAI_API_KEY}
+
+  - name: hf_minilm
+    from: huggingface:huggingface.co/sentence-transformers/all-MiniLM-L6-v2
+
+models:
+  - name: openai
+    from: openai:gpt-4o
+    params:
+      spice_tools: auto
+      openai_api_key: ${secrets:OPENAI_API_KEY}
+      system_prompt: |
+        You are an AI assistant assisting engineers with the Spice.ai OSS Project.
+
+        Always strive to be accurate, concise, and helpful in your responses.
+
+        Prefer spiceai.docs for documentation and reference information questions.
+        When generating reference links for spiceai.docs, use https://docs.spiceai.org/<docs_path> as a template and exclude `spiceaidocs/docs` prefix from docs path and `.md` file extension. Also replace `/index.md` with `/` in the path.
```

Comment:
Move these instructions to the per dataset instructions for their specific formats.

Reply:
Moved